src/overview/components/VisitAsListItem.jsx

```diff
@@ -60,20 +60,16 @@ const VisitAsListItem = ({doc, compact, onTrashButtonClick}) => {
             </div>
             <div className={styles.buttonsContainer}>
                 <button
-                    className={styles.button}
+                    className={`${styles.button} ${styles.trash}`}
```

Comment:
Better here (and in L70) just to use the `classname` package to apply multiple styles. So something like:
```
<button className={classNames(styles.button, styles.trash)} ... />
```
It really doesn't change anything though, apart from consistency with the rest of the view code.